source/features/rgh-feature-descriptions.tsx

```diff
@@ -16,13 +18,14 @@ import {isFeaturePrivate} from '../helpers/feature-utils.js';
 
 function addDescription(infoBanner: HTMLElement, id: string, meta: FeatureMeta | undefined): void {
 	const isCss = location.pathname.endsWith('.css');
+	const isCssOnly = isCss ? !$(`li[id="source/features/${id}.tsx-item"]`) : false;
```

Comment:
This is in the sidebar, right? I think we can’t expect the list to be loaded because sometimes it’s loaded later. To check this I think you can:

1. close the sidebar
2. change branch
4. visit `view-source:https://github.com/ etc etc` on the current page 

Load the source, don’t look at the Elements panel in the console. 

Anyway, this can be 
```suggestion
	const isCssOnly = isCss ? !elementExists(`li[id="source/features/${id}.tsx-item"]`) : false;
```

Also coming from `select-dom`